CHANGES.current

```diff
@@ -7,6 +7,20 @@ the issue number to the end of the URL: https://github.com/swig/swig/issues/
 Version 4.1.0 (in progress)
 ===========================
 
+2022-02-11: adr26
+            [Python] #2154 Fix memory leak.
+
+            1. SWIG python objects were being freed after the corresponding SWIG
+            module information was destroyed in Python 3, causing leaks when as
+            a result the object destructor could not be invoked. To prevent this
+            misordering, SWIG python objects now obtain a reference to the
+            Python capsule wrapping the module information, so that the module
+            information is correctly destroyed after all SWIG python objects
+            have been freed (and corresponding destructors invoked).
+
+            2. Update AppVeyor to use correct PCRE2_STATIC macro in
+            CMAKE_C_FLAGS.
```

Comment:
This file is for changes that are actually in release 4.1.0.  The CI configuration lives in the repo, but isn't part of the releases so this appveyor fix (while a good thing) doesn't seem appropriate to mention here.

This is also presumably fixing an issue introduced by the PCRE->PCRE2 change, so even if it wasn't specific to CI, it isn't actually something release 4.1.0 will fix compared to previous releases because the PCRE2 itself change will debut in 4.1.0.

Reply:
Thanks @ojwb, that makes total sense. I've merged up from master and removed the reference to the reference to the AppVeyor fix from `CHANGES.current`